src/tools/compiletest/src/header.rs

```diff
@@ -694,7 +694,6 @@ const KNOWN_DIRECTIVE_NAMES: &[&str] = &[
     "check-stdout",
     "check-test-line-numbers-match",
     "compile-flags",
-    "count",
```

Comment:
cc @jieyouxu (#121561) I presume you allowlisted `count` precisely because of `tests/rustdoc/line-breaks.rs`, am I correct?

Reply:
I must have assumed it was a valid directive too...